src/Dfc.ProviderPortal.Courses/Helpers/CosmosDbHelper.cs

```diff
@@ -128,14 +128,14 @@ public List<Models.Course> GetDocumentsByUKPRN(DocumentClient client, string col
             Uri uri = UriFactory.CreateDocumentCollectionUri(_settings.DatabaseId, collectionId);
             FeedOptions options = new FeedOptions { EnableCrossPartitionQuery = true, MaxItemCount = -1 };
 
-            List<Models.Course> docs = client.CreateDocumentQuery<Course>(uri, options)
+            List<Models.Course> docs = await client.CreateDocumentQuery<Course>(uri, options)
```

Comment:
Suggest use `var` instead of strong typed.

Reply:
Actually the whole return can be inlined :+1: 

Improving this stuff is like shooting fish in a barrel with a cannon.